docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    cand = _prune_candidates(subseqs, min_dist, cand_index)
</pre><p>To be consistent with the output of phase 2, this would be:</p><p> </p><pre class="ql-syntax" spellcheck="false">cand = np.flatnonzero(_prune_candidates(subseqs, min_dist, cand_index))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Will be fixed in next commits.